### DISCUSSION TITLE
Using podman with the macOS 14 runner

### DISCUSSION THREAD

**Discussion by cezmunsta** [actions/virtual-environments#9304] [General questions]
I am not sure if this is a limitation of the infrastructure, or whether it is something that is resolvable, but trying to use `podman` with the `macos-14` runner errors due to `QEMU`.

Using something like the following step:
```yaml
      - name: Prepare OS
        id: podman_install
        run: |
          case "${{ runner.os }}" in
            "macOS")
              brew install podman
              podman machine init
              podman machine start
              echo "DOCKER_HOST=unix://$(podman machine inspect --format '{{.ConnectionInfo.PodmanSocket.Path}}')" >> "$GITHUB_ENV"
              ;;
            "Linux")
              sudo apt update -qqy
              sudo apt install -qqy podman
              systemctl --user start podman
              echo "DOCKER_HOST=unix:///run/user/$UID/podman/podman.sock" >> "$GITHUB_ENV"
              ;;
          esac
```
The macOS runner fails with the following error:
```
Error: qemu exited unexpectedly with exit code -1, stderr: qemu-system-aarch64: -accel hvf: Error: HV_UNSUPPORTED
```
Any ideas / solutions welcome.

Thanks

o/

**Comment by mikhailkoliada** [comment]
Hey! macOS-14 runners are M1 ones, which do not support nested virtualisation

**Reply by cezmunsta** [reply]
Thanks for confirming the reason. I guess that using Homebrew to install HashiCorp Vault is the only option then, unless you have any suggestions?

**Reply by mikhailkoliada** [reply]
I am not sure that vault will work either, because it requires you to have some backend with nested virtualisation supported
